Presentation.Web/app/components/org/structure/org-structure.controller.ts

```diff
@@ -392,15 +419,20 @@
                                         ean: unit.ean,
                                         localId: unit.localId,
                                         parentId: unit.parentId,
-                                        organizationId: unit.organizationId
+                                        organizationId: unit.organizationId,
+                                        externalOriginUuid: unit.externalOriginUuid,
+                                        origin: unit.origin
                                     });
                             }
 
                             bindParentSelect($modalScope.orgUnit, orgUnits);
 
                             // only allow changing the parent if user is admin, and the unit isn't at the root
                             $modalScope.isAdmin = user.isGlobalAdmin || user.isLocalAdmin;
-                            $modalScope.canChangeParent = $modalScope.isAdmin && !$modalScope.orgUnit.isRoot;
+                            $modalScope.canChangeParent = $modalScope.isAdmin && !$modalScope.orgUnit.isRoot && !$modalScope.orgUnit.isFkOrganizationUnit;
+                            $modalScope.canChangeName = $modalScope.isAdmin && !$modalScope.orgUnit.isFkOrganizationUnit;
+                            $modalScope.supplementaryText = getSupplementaryTextForEditDialog(unit);
+                            $modalScope.canDelete = $modalScope.isAdmin && !$modalScope.orgUnit.isFkOrganizationUnit;
```

Comment:
this is just placeholder for now.. in the story about actual "rights" I will expose the available commands through the individual dto in stead of this UI magic